Applying to Pen Testing no longer "required"

TeppeiIshii-zz opened the pull request:
<Source>:
Submit Azure Service Penetration Testing Notification
https://portal.msrc.microsoft.com/en-us/engage/pentest 

This form is optional. Customers are allowed to test their Azure resources without prior approval as long as they abide by the Pentest Rules of Engagement. No exceptions to the rules of engagement will be granted.

Microsoft Cloud Unified Penetration Testing Rules of Engagement
https://technet.microsoft.com/en-us/mt784683
Customers who wish to formally document upcoming penetration testing engagements against Microsoft Azure are encouraged to fill out the Azure Service Penetration Testing Notification form.


Wording of "need to" "optional" and "encouraged to" are different, so some customers wonders what is true.  It may be better if we have the same message to all documents.

azurecla commented:
Hi __@ShawnJackson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

Teppei-MSFT committed:
Applying to Pen Testing no longer "required"

<Source>:
Submit Azure Service Penetration Testing Notification
https://portal.msrc.microsoft.com/en-us/engage/pentest 

This form is optional. Customers are allowed to test their Azure resources without prior approval as long as they abide by the Pentest Rules of Engagement. No exceptions to the rules of engagement will be granted.

Microsoft Cloud Unified Penetration Testing Rules of Engagement
https://technet.microsoft.com/en-us/mt784683
Customers who wish to formally document upcoming penetration testing engagements against Microsoft Azure are encouraged to fill out the Azure Service Penetration Testing Notification form.


Wording of "need to" "optional" and "encouraged to" are different, so some customers wonders what is true.  It may be better if we have the same message to all documents.
